src/NexusMods.Collections/CollectionCreator.cs

```diff
@@ -0,0 +1,159 @@
+using DynamicData.Kernel;
+using NexusMods.Abstractions.Collections.Json;
+using NexusMods.Abstractions.Library.Models;
+using NexusMods.Abstractions.Loadouts;
+using NexusMods.Abstractions.MnemonicDB.Attributes;
+using NexusMods.Abstractions.NexusModsLibrary;
+using NexusMods.Abstractions.NexusModsLibrary.Models;
+using NexusMods.Abstractions.NexusWebApi;
+using NexusMods.Abstractions.NexusWebApi.Types;
+using NexusMods.MnemonicDB.Abstractions;
+using NexusMods.Paths;
+using CollectionMod = NexusMods.Abstractions.Collections.Json.Mod;
+
+namespace NexusMods.Collections;
+
+public static class CollectionCreator
```

Comment:
It isn't clear what the CollectionCreator is meant to do. Does it add new collections to the app? Does it add new collections to Nexus? Does it just add new revisions to Nexus? 

Please use comments to define the responsibility scope of the class. Can also consider alternative names if they reduce ambiguousness.